sources/packages/backend/libs/sims-db/src/entities/student-assessment.model.ts

```diff
@@ -221,6 +221,15 @@ export class StudentAssessment extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   applicationOfferingChangeRequest?: ApplicationOfferingChangeRequest;
+  /**
+   * Assessment workflow data.
```

Comment:
I made a comment adding a bit more context to this in the SQL migration. If that is changed, please change it here also.